client.go

```diff
@@ -390,6 +392,27 @@ type Client struct {
 // Initialize a default *Client instance
 var DefaultClient = newClient(nil)
 
+func (c *Client) SetIgnoreErrors(errs []string) error {
+	joinedRegexp := strings.Join(errs, "|")
+	r, err := regexp.Compile(joinedRegexp)
+	if err != nil {
+		return fmt.Errorf("failed to compile regexp %q for %q: %v", joinedRegexp, errs, err)
+	}
+
+	c.ignoreErrorsRegexp = r
+	return nil
+}
+
+func (c *Client) shouldExcludeErr(errStr string) bool {
+	if c.ignoreErrorsRegexp.MatchString(errStr) {
```

Comment:
This will panic, won't it? If `c.ignoreErrorsRegexp` is never set.

https://play.golang.org/p/qGv5EKAsyX